romana-install/roles/romana/install-master/templates/config/romana.conf.yml

```diff
@@ -63,6 +63,9 @@ services:
     config:
       lease_file : "/etc/ethers"
       wait_for_iface_try: 6
+      store:
+        type: sqlite3
+        database: /var/agent.db
```

Comment:
Should this be `/var/lib/romana/agent.db` ? In top of `/var` doesn't seem right.